### DISCUSSION TITLE
Documentation and data sets

### DISCUSSION THREAD

**Discussion by lebeaupk** [carissalow/rapids#176] [General]
Hi, 
I have read your paper "Reproducible Analysis Pipeline for Data Streams: Open-Source Software to Process Data Collected with Mobile Devices" in frontiers in Digital Health (Nov 2018)" and am excited about your endeavors. However, the links to the documentation and data sets are not working (either the message is not existing or it is killed by my traditional SPAM protection software (set to mild)). Do you have anything that you could point me to that possibly works? I am curious to work with some Empatica data.
Thank you.

**Comment by JulioV** [comment]
Hi! You can check out our documentation at https://www.rapids.science/1.8/ and the latest version of our paper https://www.frontiersin.org/article/10.3389/fdgth.2021.769823

**Reply by lebeaupk** [reply]
Thank you for your reply and the latest version of the paper. Unfortunately, I am still not able to reach the www.rapids.science page on any of my devices. I am sure I am not the only one.... What do you suggest I do? See the image below:
![image](https://user-images.githubusercontent.com/100314301/155545060-26ba7de5-aba6-4ff6-b2e5-dfb634ce4ec3.png)

**Reply by JulioV** [reply]
Can you try again? There was an issue with our dns.

**Reply by lebeaupk** [reply]
I've tried again but my internet provider considers it as spam (and it's set to very low protection; i never encounter a page like this...). 
![image](https://user-images.githubusercontent.com/100314301/156423796-781171df-f73b-4192-b11f-39103a34b267.png)

**Reply by JulioV** [reply]
It's difficult for us to know why ATT is flagging the site. You could add it as an exception to their list or just build the docs locally, they all are in the `docs` folder of the repo.

These are the instructions to run the docs locally `docs/developers/documentation.md`
